modules/geolocation/unittests.py

```diff
@@ -202,6 +202,100 @@ def test_point_set_2(self):
         actual = self.locator.calculate_pixel_to_geo_mapping()
         # Test
         np.testing.assert_almost_equal(actual, expected)
+        
+        
+class TestOutputAnalysis(unittest.TestCase):
```

Comment:
Name should be function/method that is being tested (especially since there isn't supposed to be a second class).